zstacklib/zstacklib/utils/plugin.py

```
@@ -11,13 +11,15 @@
 import inspect
 import threading
 import traceback
+import warnings
 
 import log
 import ConfigParser
 
 import time
 
 from zstacklib.utils import jsonobject, http
+from zstacklib.utils.misc import ignore_exception
```

Ruff (0.8.2)

`zstacklib.utils.misc.ignore_exception` imported but unused

Remove unused import: `zstacklib.utils.misc.ignore_exception`

(F401)